br/cmd/br/fips.go

```diff
@@ -13,7 +13,6 @@
 // limitations under the License.
 
 //go:build boringcrypto
-// +build boringcrypto
```

Comment:
https://stackoverflow.com/questions/68360688/whats-the-difference-between-gobuild-and-build-directives